.rspec

```diff
@@ -1,2 +1,4 @@
 --color
 --require spec_helper
+--format d
+--order rand
```

Comment:
Add one new line at the end of file, your editor can do it for you automatically